Sources/PubNubChatComponents/Previews/Previews.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  File 2.swift
+//  
+//
+//  Created by Jakub Guz on 9/14/22.
+//
+
+import PubNub
+import PubNubChat
+import CoreData
+import Combine
+import UIKit
+import SwiftUI
+import ChatLayout
+
+#if canImport(SwiftUI) && DEBUG
+
+struct ExamplePreview: PreviewProvider {
+  
+  static var previews: some View {
+    
+    Group() {
+      
+      // Light Mode
+      UIViewPreview() {
+        let view = MessageListItemCell()
+        view.configure(MyViewModel(), theme: .incomingGroupChat)
+        return view
+      }
+      .previewLayout(.fixed(width: 414, height: 140))
+      .previewDisplayName("Light Mode")
+      .preferredColorScheme(.light)
+      
+      // Dark Mode
+      UIViewPreview() {
+        let view = MessageListItemCell()
+        view.configure(MyViewModel(), theme: .incomingGroupChat)
+        return view
+      }
+      .previewLayout(.fixed(width: 414, height: 140))
+      .previewDisplayName("Dark Mode")
+      .preferredColorScheme(.dark)
+      
+      // RTL
+      UIViewPreview() {
+        let view = MessageListItemCell()
+        view.configure(MyViewModel(), theme: .incomingGroupChat)
+        return view
+      }
+      .environment(\.layoutDirection, .rightToLeft)
+      .previewLayout(.fixed(width: 414, height: 140))
+      .previewDisplayName("RTL")
+      
+      // Accessibility
+      UIViewPreview() {
+        
+        let view = MessageListItemCell()
+        view.configure(MyViewModel(), theme: .incomingGroupChat)
+        return view
```

Comment:
UIViewPreview contains always the same view, it can be created once in function

Reply:
Additionally in that way it will be more self-describing and comments would not be necessary

Reply:
Yeah, I'm sure there are better options to solve it. I just wanted to quickly check if it's ever possible to use previews in Chat Components (it's a spike) and what are possible limitations so don't treat the current implementation very serious